parser.go

```diff
@@ -144,7 +145,7 @@ func printParsingDebug() {
 
 func parse() {
 	switch {
-	case char == ' ' || char == '\t' || char == '\n':
```

Comment:
Would the [`unicode.IsSpace`](https://pkg.go.dev/unicode#IsSpace) method be of any use for determining if a character is whitespace here and other places?

Reply:
Oh, that's neat. I will try that, thanks!